shell/platform/tizen/tizen_renderer_ecore_wl2.cc

```diff
@@ -651,31 +576,33 @@ bool TizenRendererEcoreWl2::IsSupportedExtention(const char* name) {
   return false;
 }
 
-void TizenRendererEcoreWl2::OnEnabledWlRegistryGlobalObject(
-    void* data,
-    struct wl_registry* registry,
-    uint32_t name,
-    const char* interface,
-    uint32_t version) {
-  // To use the Tizen Policy, initialize tizen_policy handle and add a listener.
-  if (strcmp(interface, tizen_policy_interface.name) == 0) {
-    uint32_t client_version = std::min(version, kMaxTizenClientVersion);
-
-    tizen_policy_ = static_cast<tizen_policy*>(wl_registry_bind(
-        registry, name, &tizen_policy_interface, client_version));
-    if (!tizen_policy_) {
-      FT_LOG(Error) << "wl_registry_bind(tizen_policy_interface) failed.";
-      return;
+void TizenRendererEcoreWl2::SetTizenPolicyNotificationlevel(int level) {
+  Eina_Iterator* itr = ecore_wl2_display_globals_get(ecore_wl2_display_);
+  struct wl_registry* registry =
+      ecore_wl2_display_registry_get(ecore_wl2_display_);
+
+  if (itr && registry) {
+    Ecore_Wl2_Global* global = nullptr;
+
+    // Retrieve global objects to bind tizen policy
+    EINA_ITERATOR_FOREACH(itr, global) {
+      if (strcmp(global->interface, tizen_policy_interface.name) == 0) {
+        tizen_policy_ = static_cast<tizen_policy*>(
+            wl_registry_bind(registry, global->id, &tizen_policy_interface, 1));
+        break;
+      }
     }
```

Comment:
'itr' may be  destroyed by eina_iterator_free

Reply:
I found the below note. Thank you.

```
Note
The caller of this function should free the returned Eina_Iterator when finished with it.
```